src/main/java/appeng/core/Registration.java

```diff
@@ -216,7 +214,6 @@ private void registerCraftHandlers( final IRecipeHandlerRegistry registry )
 		registry.addNewCraftHandler( "mekechamber", MekEnrichment.class );
 		registry.addNewCraftHandler( "grind", Grind.class );
 		registry.addNewCraftHandler( "crusher", Crusher.class );
-		registry.addNewCraftHandler( "grindfz", GrindFZ.class );
 		registry.addNewCraftHandler( "pulverizer", Pulverizer.class );
 		registry.addNewCraftHandler( "macerator", Macerator.class );
```

Comment:
It might be an idea to move the crafthandlers into their respective integration module in a later step.